### PR TITLE
Fix centernet batched operations and labels

### PR DESCRIPTION
**Pull request by schencej**

### CODE REVIEW

File: smqtk_detection/impls/detect_image_objects/centernet.py [hunk 1]
```diff
@@ -129,155 +133,220 @@ def __init__(
...
         img_iter: Iterable[np.ndarray]
     ) -> Iterable[Iterable[Tuple[AxisAlignedBoundingBox, Dict[Hashable, float]]]]:
 
-        proc_imgs = []
-        centers = []
-        trans_scales = []
-        out_height = self.input_h // 2
-        out_width = self.input_w // 2
+        img_set = _ImageDataset(img_iter, self.flip, self.scales, self._preprocess_img)
+        batch_loader = DataLoader(img_set, batch_size=self.batch_size)
```
Comment (Purg) [inline smqtk_detection/impls/detect_image_objects/centernet.py:166]: It is probably the case that we want to pass something to DataLoader's `num_workers` parameter to make use of the parallelization of data loading. The value to pass should be parameterized in the constructor.

Comment (Purg) [reply smqtk_detection/impls/detect_image_objects/centernet.py:166]: Of course, I say that, but I just saw this try to pull over 100GB of RAM so I'm trying to understand that.

**Comment by Purg** [comment]
With this being a "fix" could this please target the `release` branch instead of `master`? I would like to include this in a patch release instead of a new minor bump.

**Comment by Purg** [comment]
I think this basically looks good to me, modulo the `DataLoader` num workers parameterization.

**Review by Purg — APPROVED** [review]
LGTM. I'm not sure why CI didn't run, so I'll force-rebase with no changes to try to get it to trigger.
